CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -14,6 +14,51 @@ public function install() {
 
   public function uninstall() {
     $this->executeSqlFile('sql/uninstall.sql');
+    $this->uninstallCustomInformation();
+  }
+
+  /**
+   *  Uninstall custom information
+   */
+  private function uninstallCustomInformation() {
+    $this->deleteField('OptionGroup', 'direct_debit_codes');
```

Comment:
you can create store all the values in array then loop and call deletField instead of repeating the call.